bookmarks/views/manifest.py

```diff
@@ -5,9 +5,91 @@
 def manifest(request):
     response = {
         "short_name": "linkding",
+        "name": "linkding",
+        "description": "Self-hosted bookmark service",
         "start_url": "bookmarks",
         "display": "standalone",
         "scope": "/" + settings.LD_CONTEXT_PATH,
+        "theme_color": "#5856e0",
+        "background_color": "#ffffff" if request.user_profile.theme == "light" else "#161822",
```

Comment:
This will also use the dark color when using the default setting "auto". Maybe switch the condition to check for dark specifically.

Reply:
Done, it's now white by default, only setting when the theme is dark specifically